test/ngResource/resourceSpec.js

```diff
@@ -623,34 +623,39 @@ describe("basic usage", function() {
 
 
   it('should support dynamic default parameters (global)', function() {
-    var currentGroup = 'students',
-        Person = $resource('/Person/:group/:id', { group: function() { return currentGroup; }});
+    inject(function($q) {
```

Comment:
Nitpick: Wrap the test funnction with `inject()` and avoid the extra indentation. E.g.:

```js
it('...', inject(function($q) {...});
```